zxfer

```diff
@@ -240,7 +241,7 @@ get_os() {
 init_variables() {
   get_os "$option_T"
   dest_os="$output_os"
-  # Stupid way to detect if FreeBSD 10 through 19
+  # Stupid way to detect if FreeBSD 10 through 19 - BROKEN
   dest_osver=$(printf %.1s "$output_osver")
```

Comment:
The best way to do this is likely to use 'uname -K', which will output something like: 1101001

And you can then just do 'if [ $osver -gt 11000000 ];
for 11.0 and up